### DISCUSSION TITLE
ECONNREFUSED 127.0.0.1:30001

### DISCUSSION THREAD

**Discussion by psascue** [aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework#9] [Q&A] [answered]
Hi all, I am building a proof-of-concept and fairly new to most of these services so help is appreciated. 

I have just deployed the framework in a sandbox environment and currently following this [walkthrough](https://github.com/aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework/blob/main/docs/walkthrough.md).

I ran into this error in Postman: ECONNREFUSED 127.0.0.1:30001

Did I miss anything?

**Comment by johnrotach** [comment, marked as answer]
Hello @psascue.

Thanks for checking out this repo. From the error it appears Postman is trying to connect to localhost. Were you intending to try out an API you have deployed to the cloud? If so, there is a helper script that will generate a Postman environment based off of your deployment that you can import into Postman. Check out [this doc](https://github.com/aws-solutions-library-samples/guidance-for-aws-sustainability-insights-framework/blob/main/docs/integration/postman.md).

You could also manually update your environment with API endpoint values. You can find these in systems manager parameters with the format (`/sif/{sif tenant id}/{sif environment}/{sif module}/apiUrl`).

Please reach out if this doesn't help you get past this.

**Reply by psascue** [reply]
I completely missed that doc, thanks @johnrotach! Changing the URLs fixed it.
